bin/nexus-blobstores-create

```diff
@@ -64,12 +64,12 @@ while [[ $# -gt 0 ]]; do
             -d @-
         status=$?
         if [[ $status -eq 0 ]]; then
-            echo >&2 "INFO Successfully created ${blobstore_type} blobstore: ${blobstore_name}"
+            echo >&2 " OK created"
         else
-            echo >&2 "ERROR Failed to create ${blobstore_type} blobstore: ${blobstore_name}"
+            echo >&2 " FAIL"
 
             # Failed to create, try to update
-            echo -n >&2 "DEBUG Updating ${blobstore_type} blobstore: ${blobstore_name} as creating failed ... "
+            echo -n >&2 "Updating ${blobstore_type} blobstore: ${blobstore_name}... "
```

Comment:
IUF will not be able to recognize this message if DEBUG is removed. Also updation is done as creation has failed. This is for extra information.